airootfs/usr/share/vx-img/flash-image.sh

```diff
@@ -42,45 +69,55 @@ if [[ $_surface == 0 && $_haskeys == 0 ]]; then
     fi
 fi
 
-echo "Flashing a new image to the hard disk. This will destroy any existing data on the disk. Continue? [y/N]:"
+clear
 
-read -r answer
+data=$(lsblk -x SIZE -nblo NAME,LABEL,SIZE,TYPE | grep "Data" | awk '{ print $1 }')
 
-if [[ $answer != 'y' && $answer != 'Y' ]]; then
-    exit
-fi
+if [[ -n $data ]]; then
```

Comment:
what is this checking?

Reply:
if a disk with a "Data" label is found, `data` will be set to its name. If the disk isn't found, it won't be set. This if statement is checking whether data is set, and if it is, skipping the logic for selecting the data disk.